pages/architecture.md

```diff
@@ -207,6 +214,8 @@ As explained in the frontend section, Amazon Cognito is used for Authentication
 In Amazon API Gateway we again use [Cognito for Authorization](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-integrate-with-cognito.html). 
 With an Amazon Cognito user pool, we control who can access our GraphQL API. 
 
+**Note** - You can also deploy data.all with your own IDP. In order to do so please follow instruction in ["Deploy to AWS"](./deploy/deploy_aws.md) section 
```

Comment:
The general recommendation in "deploy data.all with your own IdP" is to keep using Cognito as a proxy. So in this paragraph I would be a bit more specific: "You can also replace Cognito authorization by a custom authorizer using your own IdP"

Reply:
Added a statement which gives clear directions to the user when they want to implement a custom authorizer. Let me know if that sounds fine.